BrainPortal/config/console_rc/lib/interactive_bourreau_control.rb

```diff
@@ -31,7 +31,7 @@
 #
 # The simplest use is to create an object for the
 # interactive session then invoke the method for the
-# command line interace:
+# command line interacted:
```

Comment:
This line was changed from "interace" (which is wrong) to "interacted", but it was actually meant to be `interface`